Add webdam resource to webguide

tenken opened the pull request:
Let me know if anything is amiss.

refs #165

tenken committed:
Sync content (#11)

* Added a note about preferring relative units

* Added link to GitHub wiki, added explanation of IT resources

* Added a note about content generated with CSS

* Added note about preferring text over images of text

* Added a note about color contrast

* Added additional notes on color blindness

tenken committed:
Document WebDam and the photoshelter archive. (#12)

garster commented on campus-resources.md:
```diff
@@ -34,14 +34,13 @@ answers, and information in support of computing at UCSB.
+you may hire [Instructional Development Photography Services](http://photo.production.id.ucsb.edu/) photographers to shoot on location or in their studio.
+
+In August 2016, a collaboration of many departments created the [UCSB Digital Asset Library](https://ucsb.webdamdb.com/
+), hosted by WebDAM, which is a resource available to all UCSB personnel and is to act as a centralized location for high quality photography of the campus. To use the service simply provide your UCSBNetID and password at login.
```
The proper case for UCSBNetID is UCSBnetID, lower case "net"

tenken committed:
Remove login details for webdam.

However we log into the system over time may change, let the login page describe that; not the webguide.

tenken commented:
Hi I have removed the spirious directions.

We should tell larger student facing services about the UCSBnetID convention -- like I dunno GOLD login prompts:

https://www.google.com/search?newwindow=1&site=&source=hp&q=%22UCSBNetID%22&oq=%22UCSBNetID%22&gs_l=hp.3..0l2j0i10i30l2j0i5i30.1179.4942.0.5912.12.12.0.0.0.0.131.1152.4j7.11.0....0...1c.1.64.hp..1.9.951.0..0i10j0i5i10i30.mj1CeFKK7Ow

garster commented:
@tenken I just go by the source, which is UCSB Identity Services http://www.identity.ucsb.edu/ and they do UCSBnetID. If other choose to not be precise, well that is on them. But I now see an error on one of the pages I control from your google search. I will need to fix it.
